CONTRIBUTING.md

```diff
@@ -32,6 +32,74 @@ Here are a few guidelines to follow when submitting a pull request:
 
 Have questions or need help with setup? Open an issue here [https://github.com/18F/web-design-standards/issues](https://github.com/18F/web-design-standards/issues).
 
+### Confirming your environment is prepared
+
+#### Confirm Ruby
+
+To serve the site locally, we will need Ruby. Confirm Ruby is installed:
+
+```sh
+gem -v
+```
+
+If you do not see a version number, go to the [RubyGems download page](https://rubygems.org/pages/download) to install Ruby.
+
+#### Confirm Ruby Bundler
+
+```sh
+bundle -v
+```
+
+If you do not see a version number, and you have confirmed Ruby:
+
+```sh
+gem install bundler
+```
+
+#### Confirm NPM
+
+To install the required modules, we will need Node Package Manager (NPM). Confirm NPM is available:
+
+```sh
+npm -v
+```
+
+If you do not see a version number, go to the [Node.js download page](https://nodejs.org/en/download/) to install Node.js.
```

Comment:
Now that I see that, I'm struggling with how to word it. The basis of what I want to say is:

We recommend you use the node version manager to install Node because it ensures you won't have to run sudo for install packages and allows you to easily switch between different node versions. You can also install directly from the Node website if nvm doesn't work for you.